<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Confiz-6090" failures="1" tests="12" name="LoginTestCases" time="41.031" errors="0" timestamp="2022-05-16T17:42:10 PKT">
  <testcase classname="automationtest.test.LoginTestCases" name="successfullUserLogin" time="3.429"/>
  <testcase classname="automationtest.test.LoginTestCases" name="Inventory" time="3.16"/>
  <testcase classname="automationtest.test.LoginTestCases" name="Cart" time="3.152"/>
  <testcase classname="automationtest.test.LoginTestCases" name="Checkout1" time="3.497"/>
  <testcase classname="automationtest.test.LoginTestCases" name="Checkout2" time="3.239"/>
  <testcase classname="automationtest.test.LoginTestCases" name="Checkout3" time="0.055"/>
  <testcase classname="automationtest.test.Verifyfalse" name="successfullUserLogin" time="3.279"/>
  <testcase classname="automationtest.test.Verifyfalse" name="Inventory" time="3.155"/>
  <testcase classname="automationtest.test.Verifyfalse" name="Cart" time="3.136"/>
  <testcase classname="automationtest.test.Verifyfalse" name="Checkout1" time="3.509"/>
  <testcase classname="automationtest.test.Verifyfalse" name="Checkout2" time="3.207"/>
  <testcase classname="automationtest.test.Verifyfalse" name="Checkout3" time="0.039">
    <failure type="java.lang.AssertionError" message="expected [rr] but found [THANK YOU FOR YOUR ORDER]">
      <![CDATA[java.lang.AssertionError: expected [rr] but found [THANK YOU FOR YOUR ORDER]
at automationtest.test.Verifyfalse.Checkout3(Verifyfalse.java:98)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Checkout3 -->
</testsuite> <!-- LoginTestCases -->
